<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnSendData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenPort">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>191</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>TCP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Serial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Modbus RTU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Modbus TCP</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnReq">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Modbus Req</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuComm">
    <property name="title">
     <string>Comm..</string>
    </property>
   </widget>
   <addaction name="menuComm"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
